src/mon/Monitor.cc

```diff
@@ -464,6 +464,7 @@ const char** Monitor::get_tracked_conf_keys() const
     "clog_to_graylog",
     "clog_to_graylog_host",
     "clog_to_graylog_port",
+    "host",
```

Comment:
Sorry, I don't verify this by myself. However I don't see a reason why we shall register all other keys but leaving "host" aside, which is weird:)
Anyway,  would you like me to drop or keep this?